vantage6-client/vantage6/client/subclients/algorithm_store.py

```
@@ -0,0 +1,182 @@
+from vantage6.client.filter import post_filtering
+from vantage6.common.client.client_base import ClientBase
+
+
+class AlgorithmStoreSubClient(ClientBase.SubClient):
+    """Subclient for the algorithm store."""
+
+    def __init__(self, parent: ClientBase):
+        super().__init__(parent)
+        self.store_url = None
+
+    def set(self, id_: int) -> dict:
+        """ "
+        Set the algorithm store to use for the client.
+
+        Parameters
+        ----------
+        id_ : int
+            The id of the algorithm store.
+
+        Returns
+        -------
+        dict
+            The algorithm store.
+        """
+        store = self.get(id_)
+        if "url" in store:
+            self.store_url = store["url"]
```

Is the `store_url` optional or should we report an error here?